backend/src/modules/Boards/repository/inMemory/InMemoryBoards.repository.js

```diff
@@ -0,0 +1,15 @@
+export class InMemoryBoardsRepository {
+  constructor() {
+    this.entities = {};
+  }
+  async createNewBoard(boardDetails) {
+    const nameAndCoords = `${boardDetails.name}+ ${boardDetails.coords}`;
```

Comment:
Błędu tu nie ma, ale zastanawiam się czy na pewno taki nameAndCord chciałaś uzyskać z tego templateString  ```suggestion
    const nameAndCoords = `${boardDetails.name}+ ${boardDetails.coords}`;
```

Reply:
Napisz proszę test do InMemory na:
dodawanie i znajdowanie elementu
(albo 2 testy osobno na dodawanie i osobno na dodawanie i znajdowanie.